bgpd/bgp_attr.c

```diff
@@ -2506,6 +2506,12 @@ static bgp_attr_parse_ret_t bgp_attr_psid_sub(uint8_t type, uint16_t length,
 		}
 
 		/* Configure from Info */
+		if (attr->srv6_vpn) {
```

Comment:
> Malformed attribute with repeated SRv6 VPN or L3VPN fields would parse as correct, and leak 32 bytes for each excess copy

Ah, that's right. 

I check the draft that is talking about "if multiple same-TLV are presented".

Just a moment pls